src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx

```diff
@@ -201,9 +203,21 @@ export class OrganizationIntegrations extends AsyncComponent<
   }
 
   sortIntegrations(integrations: AppOrProviderOrPlugin[]) {
-    return integrations
-      .sort((a, b) => a.name.localeCompare(b.name))
-      .sort((a, b) => this.getInstallValue(b) - this.getInstallValue(a));
+    const sortByName = arr => arr.sort((a, b) => a.name.localeCompare(b.name));
+    const sortByPopularity = arr =>
+      arr.sort((a, b) => {
+        const weightA = weights[a.slug] ? weights[a.slug] : 1;
+        const weightB = weights[b.slug] ? weights[b.slug] : 1;
```

Comment:
nit: `const weightA = weights[a.slug] || 1` is a bit shorter (assuming weights are never 0, in which case it would be falsey and mess this code up.)

Reply:
But keep in mind, if the weights are ever `0` both of these would mess up, and use `1`

Reply:
@leeandher Maybe we can do:
`Number.isInteger(weights[a.slug]) ? weights[a.slug] : 1;`

It should work for 0.

Reply:
You want [the null coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)